src/scales/scale.ts

```diff
@@ -22,11 +22,11 @@ module Plottable {
      * If one end of the domain is set to an excepted value as a result of autoDomain()-ing,
      * that end of the domain will not be padded.
      *
-     * @param {Scale} scale
-     * @returns {D[]} An array of extents.
+     * @param {QuantitativeScale} scale
+     * @returns {D[]} An array of values to become padding exceptions.
```

Comment:
What do you mean? They *are* not exceptions, unless the providers provide them, hence they *become* exceptions